server/routes/induction/update/workedBeforeFormValidator.ts

```diff
@@ -0,0 +1,24 @@
+import type { WorkedBeforeForm } from 'inductionForms'
+import type { PrisonerSummary } from 'viewModels'
+import formatErrors from '../../errorFormatter'
+
+export default function validateWorkedBeforeForm(
+  workedBeforeForm: WorkedBeforeForm,
+  prisonerSummary: PrisonerSummary,
+): Array<Record<string, string>> {
+  const errors: Array<Record<string, string>> = []
+
+  errors.push(...formatErrors('workedBefore', validateWorkedBefore(workedBeforeForm, prisonerSummary)))
```

Comment:
The first arg is the name of the field on the form/page that the error relates to (so the nunjucks macro can find it on the page and highlight it red etc)
In this case is should be `hasWorkedBefore`

IE: 
```
errors.push(...formatErrors('hasWorkedBefore', validateWorkedBefore(workedBeforeForm, prisonerSummary)))
```